detect_ai_content/ml_logic/for_texts/lgbm.py

```diff
@@ -0,0 +1,86 @@
+import pandas as pd
+import os
+import pickle
+from create_internal_features import InternalFeatures
+
+class LightGbm:
+
+    def local_trained_pipeline(self):
+        import detect_ai_content
+        module_dir_path = os.path.dirname(detect_ai_content.__file__)
+        model_path = f'{module_dir_path}/../detect_ai_content/models/leverdewagon/{self.mlflow_model_name}_pipeline.pickle'
+        return pickle.load(open(model_path, 'rb'))
+
+    def load_model():
+
+        """
+        - Return a model
+        - Return None (but do not Raise) if no model is found
+        """
+
+        model= None
+
+        return model
+
+    def __init__(self):
+        self.description = "XgBoost model 13 nov updated"
```

Comment:
LightGbm is same as Xgboost ?

Reply:
Nope... i think Yuka told yesterday its not ready.